artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -327,6 +327,9 @@ public String[] getStaticNodes() {
    @Option(names = "--jdbc-lock-expiration", description = "Lock expiration (in milliseconds).")
    long jdbcLockExpiration = ActiveMQDefaultConfiguration.getDefaultJdbcLockExpirationMillis();
 
+   @Option(names = "--aio-journal-max-io", description = "The journal-max-io value to use when also using the ASYNCIO journal-type. When using NIO or MAPPED this value is always '1'. Default: 4096")
```

Comment:
what about --journal-max-aio


(journal max asynchronous io).

Reply:
That's not bad!